### PR TITLE
Added a new tile 

### PR DESCRIPTION
**Pull request by samyakbvs**
I and @arjunpandey2003 collaborated on this PR to add a new tile to the game

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
Collabration of @samyakbvs and @arjunpandey2003

**Comment by arjunpandey2003** [comment]
@duttaditya18 since we have collaborated whats the problem, actually as you know about my laptop 
there is no Inkscape so we collaborated in order to make this successful.

**Comment by samyakbvs** [comment]
@duttaditya18 I am helping out Arjun as his laptop has broken so he can't install Inkscape and make a tile himself

**Comment by arjunpandey2003** [comment]
@niccokunzmann pls review and merge thanks

File: js/levels.js [hunk 1]
```diff
@@ -8,39 +8,51 @@
 
 // createTestLevel is used in /js/gui.js ignoring the error
 // ignore JSHintBear
-function createTestLevel(level) {
-  if(level == 2) {
-    /* 2nd level */
-    return new Level([
```
Comment (niccokunzmann) [inline js/levels.js]: Please tell me why you removed this code. It allows loading different levels.

**Comment by niccokunzmann** [comment]
@samyakbvs Thanks, I reviewed. What do you think?

**Comment by arjunpandey2003** [comment]
@niccokunzmann i think @samyakbvs did those changes will check might have done by mistake

**Comment by arjunpandey2003** [comment]
@niccokunzmann done

**Comment by samyakbvs** [comment]
@niccokunzmann Ya I did those by mistake

**Comment by niccokunzmann** [comment]
I can merge this. Please remember if you make formatting changes, this affects all people who edit the file and they need to merge again.

**Comment by arjunpandey2003** [comment]
thanks @niccokunzmann

**Pull request merged**